### ISSUE TITLE
Add intelligent `modules()` method

### ISSUE THREAD

**Issue by medienbaecker** [medienbaecker/kirby-modules#22]
It would be great to have a `modules()` method which would work like the `renderModules()` method but also allow loops.

```php
<?= $page->modules() ?>
```

```php
<?php
foreach ($page->modules() as $module) {
  // Do something with the module
}
?>
```

**Comment by pedroborges** [comment]
<img width="1351" alt="Screen Shot 2020-09-29 at 18 41 03" src="https://user-images.githubusercontent.com/688309/94619476-5d0c7180-0283-11eb-9f7c-38e2318ccf2d.png">

I did a quick proof-of-concept and it works. Kirby has the `Kirby\Cms\Pages` collection class. I created a basic `ModulesCollection` class that extends `Pages` and implements the `toString` method by rendering all modules. `toArray()` is already taken care of by the `Pages` class.

Creating a `modules` page method that returns an instance of `ModulesCollection` would do the trick.

Let me know if you would be interested in a PR.
